### DISCUSSION TITLE
Request access to repository settings

### DISCUSSION THREAD

**Discussion by Salkimmich** [gramineproject/gramine#1781] [Ideas]
Hello Gramine Maintainers:

I'm reaching out to request access to the repository settings for Gramine. Many best practices have already been established within the project, but I am preparing for future updates and enhancements to further strengthen security, compliance, and collaboration efforts.

To effectively review and enhance the project's security, compliance, and collaboration efforts, I require ownership or administrative privileges within the repository. This level of access will enable me to conduct a thorough review, identify areas for improvement, and propose enhancements that align with the project's objectives. Once this review is complete, you are welcome to remove these privileges, and alternatively, I can help you to establish these by assisting your administrators with direct implementation.

I would like to run [CLOMonitor](https://clomonitor.io/search?page=1) checks for each repository, but requesting direct access to repository settings so I can run these checks locally without exposing your project to a publicly available dashboard.

Granting me access to the repository settings will enable me to review the existing configurations, identify any areas for improvement, and implement additional best practices and other measures to enhance the development workflow and project governance. Best practices can be highly specific to each project's unique setup and requirements, so our approach will be flexible and tailored to accommodate the Gramine's specific needs.

Following review, any proposed changes to the process will be discussed transparently with the project maintainers and contributors. The standard PR and Issue process will be used to ensure full visibility and collaboration on all recommendations. 

I look forward to collaborating with you for future updates!

Thanks,
Sal 
CCC Technical Community Architect

**Comment by mkow** [comment]
> To effectively review and enhance the project's security
> [...]
> I require ownership or administrative privileges within the repository

No offense, but nope :) We're a security-oriented project and we don't even know you yet. I'd even say, that not giving admin/owner access to people we don't know is a part of the security best practices ;) Even most of our maintainers don't have admin access, because they don't need it (and thus they shouldn't have it).

Anyways, if you have any concrete suggestions what we could change then please let us know (with an explanation why this would help in improving our security).

Best,
Michał
